org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -1273,6 +1273,9 @@ private URI asURI(String formatterUrl) {
 		URI uri = null;
 		try {
 			uri = new URI(ResourceUtils.toClientUri(formatterUrl));
+			if (!uri.isAbsolute()) {
+				uri = new File(formatterUrl).getAbsoluteFile().toURI();
```

Comment:
If formatterUrl is set to an absolute file path, this works well. 

But if formatterUrl is set to a relative file path, i'm not sure whether `new File(formatterUrl).getAbsoluteFile()` could get the absolute path well. For this case, it seems safer to use `findFile(formatterUrl)` to get the correct absolute path.

Reply:
Fixed.